app/src/main/java/org/mozilla/fenix/home/recentvisits/controller/RecentVisitsController.kt

```diff
@@ -60,6 +60,11 @@ interface RecentVisitsController {
      * @param highlightUrl Url of the [RecentHistoryHighlight] to remove.
      */
     fun handleRemoveRecentHistoryHighlight(highlightUrl: String)
+
+    /**
+     * Called when opening the dropdown menu on a recent visit by long press.
```

Comment:
```suggestion
     * Callback for when the user user long clicks on a recent visit.
```

This is different from the above because I am mostly following the existing wording in the file. Generally, we just want to be consistent with the file.